test/lazperf_tests.cpp

```diff
@@ -1417,15 +1417,15 @@ TEST(lazperf_tests, dynamic_can_do_blind_compression) {
 
         time_t seed = time(NULL);
         srand(seed);
-
+		int rvalue = rand();
```

Comment:
I'll just warn that if the existing code wasn't working, it *may* be an indication that something really is wrong.  Been there.